staging/src/k8s.io/client-go/util/cert/cert.go

```diff
@@ -39,10 +39,11 @@ const duration365d = time.Hour * 24 * 365
 
 // Config contains the basic fields required for creating a certificate
 type Config struct {
-	CommonName   string
-	Organization []string
-	AltNames     AltNames
-	Usages       []x509.ExtKeyUsage
+	CommonName         string
+	Organization       []string
+	AltNames           AltNames
+	Usages             []x509.ExtKeyUsage
+	PublicKeyAlgorithm x509.PublicKeyAlgorithm
```

Comment:
@rojkov i pinged the #sig-api-machinery channel about this change and it is not approved.
https://kubernetes.slack.com/archives/C0EG7JC6T/p1582187786382200

i guess the alternative is to pass the algorithm type as an argument to functions that need it.

Reply:
Ok! I'll follow @rosti's suggestion to introduce a wrapper struct.